config/sewing-kit/index.ts

```diff
@@ -2,42 +2,37 @@ import {
   Package,
   createComposedProjectPlugin,
   createProjectTestPlugin,
-  createProjectBuildPlugin,
-} from '@sewing-kit/plugins';
-import {react} from '@sewing-kit/plugin-react';
-import {javascript, updateBabelPreset} from '@sewing-kit/plugin-javascript';
-import {typescript} from '@sewing-kit/plugin-typescript';
+} from '@sewing-kit/core';
+import {babel} from '@sewing-kit/plugin-babel';
 import {packageBuild} from '@sewing-kit/plugin-package-build';
 import {} from '@sewing-kit/plugin-jest';
 
-import {addLegacyDecoratorSupport} from './plugin';
-
 export function quiltPackage({
   jestEnv = 'jsdom',
-  useReact = false,
   jestTestRunner = 'jest-circus',
+  polyfill = true,
```

Comment:
The only package that overrides this and sets it to false is `polyfills` because we want that package to uh, provide the polyfills.